### PR TITLE
fix error in discover-repos.rb

### PR DESCRIPTION
**Pull request by RuslanGaliullin**
solves issue #149

### CODE REVIEW

**Comment by RuslanGaliullin** [comment]
@yegor256

**Comment by yegor256** [comment]
@RuslanGaliullin something is wrong with the test, if didn't catch this bug earlier. Can you try to fix the test first please?

**Comment by RuslanGaliullin** [comment]
May you run the action pls @yegor256

File: tests/steps/test-discover-repos.sh [hunk 1]
```diff
@@ -30,12 +30,9 @@ tex=${TARGET}/foo.tex
 
 {
     rm -f "${csv}"
-    msg=$("${LOCAL}/steps/discover-repos.rb" --dry --pause=0 --total=3 --min-stars=100 --max-stars=1000 "--csv=${csv}"  "--tex=${tex}")
+    msg=$("${LOCAL}/steps/discover-repos.rb" --dry --pause=0 --total=102 --min-stars=100 --max-stars=1000 "--csv=${csv}"  "--tex=${tex}")
     echo "${msg}"
-    echo "${msg}" | grep "Found 1 repositories in page #0"
```
Comment (yegor256) [inline tests/steps/test-discover-repos.sh:35]: @RuslanGaliullin in general, it's not a good practice to delete test assertions. You are weakening the test, which makes your pull request look suspicious: maybe you break some existing functionality? Would be better if you _add_ a new test scenario below the existing one.

Also, I think that the download of 102 repositories will make the test rather slow. Maybe it's better to decrease the size of the page to, say, 10: `--total=12 --page-size=10`

**Comment by yegor256** [comment]
@RuslanGaliullin see my comment above

File: steps/discover-repos.rb [hunk 2]
```diff
@@ -121,6 +121,7 @@
 path = File.expand_path(opts[:csv])
 FileUtils.mkdir_p(File.dirname(path))
 lines = [found.first[1].keys.join(',')] + found.values.map { |m| m.values.join(',') }
+# lines = [found.first[1].keys.join(',')] + found.map { |pair| pair[1].values.join(',') }
```
Comment (yegor256) [inline steps/discover-repos.rb:124]: @RuslanGaliullin it's better to delete the code instead of commenting it out

**Pull request closed without merging**